src/components/SubjectLoading.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SubjectLoading = ({ loaded }) => {
+  const transform = !loaded ? 'scale(1)' : 'scale(5)';
```

Comment:
Question: what's with the scaling here? Is it that, once the Subject Image has loaded, the "Subject Viewer Fit Image" action triggers and the Subject zooms out, and we need to compensate for that? If yes, it seems like we need another solution for it.

In any case, we should probably add a note here to clarify what's going on.